sites/public/pages/applications/start/choose-language.tsx

```diff
@@ -128,6 +128,17 @@ export default () => {
                   開始
                 </Button>
               )}
+
+              {listing?.applicationConfig.languages.some((lang) => lang == "zh") && (
```

Comment:
Maybe will be better to save lang codes as enum and use it for application conductor and here, what do you think?